lib/rules/font-weight-notation/index.js

```diff
@@ -17,11 +16,13 @@ const report = require('../../utils/report');
 const ruleMessages = require('../../utils/ruleMessages');
 const setDeclarationValue = require('../../utils/setDeclarationValue');
 const validateOptions = require('../../utils/validateOptions');
+const { assertString } = require('../../utils/validateTypes');
 
 const ruleName = 'font-weight-notation';
 
 const messages = ruleMessages(ruleName, {
 	expected: (type) => `Expected ${type} font-weight notation`,
+	expectedWithActual: (actual, expected) => `Expected "${actual}" to be "${expected}"`,
```

Comment:
[note] Add `expectedWithActual` for backward compatibility.

Reply:
You're right to add this.

When we first built the rule, it should have only checked `normal` and `bold` rather than relative words too. Now we have an odd mix of the `named-where-possible` and `numeric` primary options along with the `ignore: ["relative"]` secondary option.

It's too late to change it now.

Adding `expectedWithActual` is the best way to deal with backwards compatibility for things like:

https://github.com/stylelint/stylelint/blob/9abc4353de642f7274b1e87ed1211675235b9b79/lib/rules/font-weight-notation/__tests__/index.js#L175-L183